### DISCUSSION TITLE
Sergen tool error connection string

### DISCUSSION THREAD

**Discussion by minhhungit** [serenity-is/Serenity#6471] [Help]
It looks since version 6.0.0 => 6.0.8, sergen tool has error relate to connection string

![image](https://user-images.githubusercontent.com/2279508/182117624-d57b1091-fd50-4b1c-be08-f92586422415.png)

But it works with version 5.2.4

Not sure if other guys has this error

**Comment by minhhungit** [comment]
add TrustServerCertificate=True to connectionstring looks resolved it

**Comment by minhhungit** [comment]
This connection string work:
```
"ConnectionString": "Server=JINPC;Database=Demo_Default_v1;Integrated Security=true;TrustServerCertificate=True",
```

**Reply by stixoffire** [reply]
Or you could change your connection string to not integrated  ..

**Reply by minhhungit** [reply]
I remember I tried " not integrated" case but not work
